### PR TITLE
Kexugit beijing

### PR DESCRIPTION
**Pull request by kexugit**
**pull this**

### CODE REVIEW

File: kexugit-beijing.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Beijing
+
+### Places to go
+- The forbidden city
+
+### Food to eat
+- Roast duck
```
Comment (Minewiskan) [inline kexugit-beijing.md]: Do you have another recommendation for vegetarians ❓

File: kexugit-beijing.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Beijing
+
+### Places to go
+- The forbidden city
+
+### Food to eat
```
Comment (ghost) [inline kexugit-beijing.md:6]: Do you have another recommendation for vegetarians ❓

**Comment by kexugit** [comment]
pull comment

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]
Looks good

**Comment by kexugit** [comment]
merging kexugit-beijing

**Pull request merged**